tasks/sep/006-2-foundation-guardian-updates/README.md

```diff
@@ -0,0 +1,27 @@
+# Sepolia Guardian Changes - Foundation Runbook
+
+Status: READY TO SIGN
+
+## Objective
+
+This is the sepolia playbook for reinitializing the SuperchainConfig with the Security Council as Guardian, instead of the Optimism Foundation.
+This is a requirement for reaching Stage 1.
+
+## Simulation
+
+Please see the "Simulating and Verifying the Transaction" instructions in [SINGLE.md](../../../SINGLE.md).
+When simulating, ensure the logs say `Using script Using script /your/path/to/superchain-ops/tasks/sep/006-1-foundation-guardian-updates/SignFromJson.s.sol`.
+Thus ensures all safety checks are run. If the default `SignFromJson.s.sol` script is shown
+(without the full path), something is wrong and the safety checks will not run.
+
+Do NOT yet proceed to the "Execute the Transaction" section.
+
+## State Validations
+
+Please see the instructions for [validation](./VALIDATION.md).
+
+## Execution
+
+At this point you may resume following the execution instructions in the "Execute the Transaction" section of [SINGLE.md](../../../SINGLE.md).
+
+When executing, ensure the logs say `Using script Using script /your/path/to/superchain-ops/tasks/sep/006-1-foundation-guardian-updates/SignFromJson.s.sol`. Thus ensures all safety checks are run. If the default `SignFromJson.s.sol` script is shown (without the full path), something is wrong and the safety checks will not run.
```

Comment:
```suggestion
When simulating, ensure the logs say `Using script /your/path/to/superchain-ops/tasks/sep/006-2-foundation-guardian-updates/SignFromJson.s.sol`.
This ensures all safety checks are run. If the default `SignFromJson.s.sol` script is shown (without the full path), something is wrong and the safety checks will not run.
```

Reply:
I think "executing" is correct here because were in the Executing section. But agree with the other suggestions